ccp/lib/email.py

```diff
@@ -54,7 +52,7 @@ def email(self,
             to_addresses=to_addresses)
 
         # send email
-        stdout, stderr = run_cmd2(command, shell=True)
+        stdout, stderr = run_command(command, shell=True)
```

Comment:
This is major breakage!

The `run_cmd2` is supposed to return `stdout and stderr`. You have updated the `run_command` to return only one value.

There are lines in code where run_cmd is used differently, at places only one value is expected while 2 at some other places.
if you are refactoring that, I'd suggest we use the `stdout` and `stderr` approach and update all the references of `run_command` to work with two values.

Reply:
Ack fixed